src/init.cpp

```diff
@@ -297,9 +297,10 @@ void Shutdown(NodeContext& node)
     StopTorControl();
 
     // After everything has been shut down, but before things get flushed, stop the
-    // scheduler and load block thread.
-    if (node.scheduler) node.scheduler->stop();
+    // load block thread and the scheduler. Load block is stopped first, since it can
+    // call LimitValidationInterfaceQueue() which would block indefinitely without the scheduler.
```

Comment:
True. I changed the wording of the comment / commit message, referring to `SyncWithValidationInterfaceQueue` instead of `LimitValidationInterfaceQueue`.